### PR TITLE
Document need for Python 3.6

### PR DESCRIPTION
**Pull request by ivoanjo**
I ran into:

```python
    print(f"Sampling every {sample_period:,} CPU cycles")                                                                                                  
                                                       ^                                                                                                   
SyntaxError: invalid syntax 
```

on Python 3.7 due to the use of [f-strings](https://docs.python.org/3/whatsnew/3.8.html).

### CODE REVIEW

**Comment by javierhonduco** [comment]
Mmm interesting! Seems like [f-strings were added in 3.6](https://www.python.org/dev/peps/pep-0498/)

```
$ python3.7 # Python 3.7.5+ (default, Jan 24 2020, 11:19:43)
>>> sample_period = 100000
>>> print(f"Sampling every {sample_period:,} CPU cycles")
Sampling every 100,000 CPU cycles
```

Could you please try printing the Python version in case it's using some other interpreter binary?

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 rbperf is an experimental Ruby Profiler that runs in Linux's eBPF VM (extended Berkeley Packet Filters)
 
 ## Installation
-You'll need [libbcc installed](https://github.com/iovisor/bcc/blob/master/INSTALL.md) in your system and a recent Linux kernel.
+You'll need [libbcc installed](https://github.com/iovisor/bcc/blob/master/INSTALL.md) in your system, a recent Linux kernel, and Python 3.8+.
```
Comment (javierhonduco) [inline README.md]: s/3.8/3.6 😄

**Comment by ivoanjo** [comment]
Hey! D'oh, you're absolutely right.



I was trying this on a machine that had an older system python 3 (Ubuntu 16.04) and I was using [pyenv](https://github.com/pyenv/pyenv) to use a newer Python, but in combination with sudo I got the wrong python version (got the system Python 3.5) which was what got me confused.

I've corrected as suggested :)

Thanks for the attention, and really interesting work btw!

**Comment by ivoanjo** [comment]
I've just signed the CLA as well, so hopefully that will propagate soon.

**Comment by javierhonduco** [comment]
Happens to me all the time! Thanks for the PR, and let me know if I can help with anything else :)

**Pull request merged**